### PR TITLE
[FIX] pep8 line length to 80

### PR DESCRIPTION
**Pull request by nbessi**
activate pep8 check in travis, and add no qa on scenario while waiting improvement of quality-tools.
This is a PEP8 commit only, underling code quality will not be improved in this PR

### CODE REVIEW

File: account_credit_control/scenarios/features/steps/account_credit_control.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
+# flake8: noqa
```
Comment (bwrsandman) [inline account_credit_control/scenarios/features/steps/account_credit_control.py:2]: Please add a solid reason in comments when adding noqa.

A better solution would be to propose an exception to the OCA/maintainer-quality-tools linter such as "ignore rule `F401` on files a path like `*/scenarios/features/steps/*`".

File: async_move_line_importer/model/move_line_importer.py [hunk 2]
```diff
@@ -162,13 +184,16 @@ def format_messages(self, messages):
         res = []
         for msg in messages:
             rows = msg.get('rows', {})
-            res.append(_("%s. -- Field: %s -- rows %s to %s") % (msg.get('message', 'N/A'),
-                                                                 msg.get('field', 'N/A'),
-                                                                 rows.get('from', 'N/A'),
-                                                                 rows.get('to', 'N/A')))
+            res.append(_("%s. -- Field: %s -- rows %s to %s") % (
+                msg.get('message', 'N/A'),
```
Comment (bwrsandman) [inline async_move_line_importer/model/move_line_importer.py:188]: Do you think `'N/A'` should be translated? For example, some localizations only have it in minuscule `n/a` and some only in caps.

File: currency_rate_update/currency_rate_update.py [hunk 3]
```diff
@@ -721,11 +785,15 @@ def get_updated_currency(self, currency_array, main_currency, max_delta_days):
...
-                    %s. Please check provider data format and/or source code." % curr)
-                raise osv.except_osv('Error !', 'Exchange data format error for\
-                    Bank of Canada - %s !' % str(curr))
+                _logger.error(
+                    "Exchange data format error for Bank of Canada -"
+                    "%s. Please check provider data format "
+                    "and/or source code." % curr)
+                raise osv.except_osv('Error !',
+                                     'Exchange data format error for\
+                                     Bank of Canada - %s !' % str(curr))
```
Comment (bwrsandman) [inline currency_rate_update/currency_rate_update.py:797]: These strings should be translated

File: currency_rate_update/currency_rate_update.py [hunk 4]
```diff
@@ -721,11 +785,15 @@ def get_updated_currency(self, currency_array, main_currency, max_delta_days):
...
             else:
-                _logger.error("Exchange data format error for Bank of Canada -\
-                    %s. Please check provider data format and/or source code." % curr)
-                raise osv.except_osv('Error !', 'Exchange data format error for\
-                    Bank of Canada - %s !' % str(curr))
+                _logger.error(
+                    "Exchange data format error for Bank of Canada -"
+                    "%s. Please check provider data format "
+                    "and/or source code." % curr)
+                raise osv.except_osv('Error !',
```
Comment (bwrsandman) [inline currency_rate_update/currency_rate_update.py:795]: orm.except_orm

**Comment by bwrsandman** [comment]
Thank you for this work, I know it's tedious.

Aside from the `#noqa` comments, nothing is blocking.

File: account_credit_control/scenarios/features/steps/account_credit_control.py (hunk 1 above)
Comment (nbessi) [inline account_credit_control/scenarios/features/steps/account_credit_control.py:2]: Please read the first comment :P,

This has already been discussed and is waiting a cleaner solution https://github.com/OCA/maintainer-quality-tools/issues/66.

This change set will should be reverted in a timely manner.

Nicolas

**Comment by nbessi** [comment]
Tho I made a wrong rebase ... Needs fixing

File: account_credit_control/scenarios/features/steps/account_credit_control.py (hunk 1 above)
Comment (bwrsandman) [inline account_credit_control/scenarios/features/steps/account_credit_control.py:2]: Oh, I am sorry, I went directly to the code.
Can we assume this PR is on hold until the quality tools take care of this?

**Comment by nbessi** [comment]
Ok ready to merge

**Comment by pedrobaeza** [comment]
:+1: 

@bwrsandman's comments are a good improve, but maybe for other PR.

**Pull request merged**